src/libraries/System.Diagnostics.Process/src/System/Diagnostics/Process.cs

```diff
@@ -1197,8 +1198,15 @@ private void SetProcessId(int processId)
         ///       component.
         ///    </para>
         /// </devdoc>
+        [UnsupportedOSPlatform("ios")]
+        [UnsupportedOSPlatform("tvos")]
         public bool Start()
         {
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Create("IOS")) || RuntimeInformation.IsOSPlatform(OSPlatform.Create("TVOS")))
```

Comment:
Can we do this in https://github.com/dotnet/runtime/blob/79ae74f5ca5c8a6fe3a48935e85bd7374959c570/src/libraries/System.Diagnostics.Process/src/System/Diagnostics/Process.Unix.cs#L353 instead?  Also, can it use OperatingSystem.IsIOS / OperatingSystem.IsTvOS?

Reply:
Done, thanks